projects/ROCKNIX/packages/linux/package.mk

```diff
@@ -326,18 +326,26 @@ makeinstall_target() {
   rm -f ${INSTALL}/$(get_kernel_overlay_dir)/lib/modules/*/build
   rm -f ${INSTALL}/$(get_kernel_overlay_dir)/lib/modules/*/source
 
-  if [ "${BOOTLOADER}" = "u-boot" -o "${BOOTLOADER}" = "arm-efi" ]; then
+  if [ "${BOOTLOADER}" = "arm-efi" ]; then
+    mkdir -p ${INSTALL}/usr/share/bootloader/boot/grub
+    for dtb in arch/${TARGET_KERNEL_ARCH}/boot/dts/**/*.dtb; do
```

Comment:
Typo double **?

Reply:
no that's to look in multiple level of subfolder in one clobber